lib/bulbs-elements/store/store.js

```diff
@@ -79,20 +74,4 @@ function dispatchFunction (store, action, payload) {
   }
 
   deliverSubscriptions(store);
-
-  if (DEBUG) {
-    console.groupCollapsed(
-      `DISPATCH %c${action.type} %c=> %c${action.fieldName}`,
-      'color:green',
-      'color:auto',
-      'color:blue'
-    );
-    console.timeEnd(`dispatch ${action.type}`);
-    console.log('PAYLOAD: ', payload);
-    console.log('PREV VALUE: ', priorFieldValue);
-    console.log('NEXT VALUE: ', nextFieldValue);
-    console.log('PREV STORE: ', priorState);
-    console.log('NEXT STORE: ', nextState);
-    console.groupEnd();
-  }
```

Comment:
Could use a double check, the production build will strip out conditionals that always evaluate to false. If we come up with a way to set DEBUG to true at will in development we can leave this in place.